src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -91,18 +97,18 @@ def load_mlflow_model(
             raise UserConfigValidationException(
                 "Unable to retrieve model by model id {} in workspace {}, error:\n{}".format(
                     model_id, workspace.name, e
-                )
+                ), e
```

Comment:
why is this needed? ` e`

Reply:
It should have been a userconfig error to use the second e for logging the traceback object of original exception. Fixed!